src/lib.rs

```diff
@@ -675,6 +686,9 @@ impl State {
             });
 
         let full_output = self.context.end_frame();
+
+        self.egui_state.handle_platform_output(window, &self.context, full_output.platform_output);
```

Comment:
I don't think this actually fixes the clipboard on web, since egui-winit has no way of interacting with the browser clipboard - you have to handle copy/cut/paste events manually (which is also what eframe does), see my latest few commits

Reply:
On my machine it does fix the issue, though it's just a simulated clipboard in egui

Reply:
It doesn't seem like your solution works?

Reply:
We picked out your changes to copy egui's clipboard to the browser, that should do it.

Egui on its own can handle the internal clipboard just fine, so it should be alright.

Reply:
We added the paste event back.